react-backend/client/src/components/visitors/qrcode.js

```diff
@@ -93,6 +63,41 @@ export default class QRCode extends Component {
       this.props.history.push('/visitor/qrerror');
     }
   }
+  instascan = () => new Promise((resolve, reject) => {
+
+    const scanner = new Instascan.Scanner({
+      video: this.previewDiv,
+      scanPeriod: 5,
+    });
+
+    scanner.addListener('scan', (content) => {
+      if (this.previewDiv) {
+        scanner
+          .stop()
+          .then(() => {
+            resolve(content);
+          })
+          .catch(() => {
+            reject('failure stopping scanner');
+          });
+      } else {
+        this.previewDiv.play();
```

Comment:
If `previewDiv` is `null` we still try to call a method on it? This is what caused the original bug.